### ISSUE TITLE
ideabox: direct links to files via doi

### ISSUE THREAD

**Issue by lnielsen** [inveniosoftware/invenio-app-rdm#66]
Provide direct links to files through DOIs + part identifiers (like 10.2097/ASDF#prevew.png)

via Kai

**Comment by tmorrell** [comment]
Content negotiation would also be great.  So 10.2097/ASDF with image/png in the header would go to the file instead of the landing page.

**Comment by lnielsen** [comment]
Yep, essentially I think that's the only way to implement it. Problem is if a record has say 3 PDF files, then with content negotiation, you have to select one. 

I think there was some discussions on defining e.g. a format for data repositories based on bagit or similar so that this would be the way to get all files for a DOI.

**Comment by tmorrell** [comment]
Yeah, multiple files with the same type are a problem. But, better a partial solution than nothing. A bagit option for all records would be great as well.

**Comment by Herrner** [comment]
As a further reference of what I'm thinking about:  
- [ePIC part identifiers](https://doc.pidconsortium.eu/guides/api-partial/)
- [Example for the TIB Video portal](https://doi.org/10.5446/42623#t=17:50,21:55) (not useful for what I'm suggesting, but an actual implementation of PIs for DOIs)

**Comment by Herrner** [comment]
Yet another input to the (stale) topic:

[Support for content URLs](https://datacite.org/assets/Open_Hours%20-%20Support%20for%20content%20URLs.pdf) (from Datacite open hours November 2021)
